src/tests/test_dynamic_api/test_serializers.py

```diff
@@ -234,7 +236,7 @@ def test_dataset_url_prefix(
             "_links": {
                 "schema": "https://schemas.data.amsterdam.nl/datasets/test/afvalwegingen/afvalwegingen#containers",  # noqa: E501
                 "self": {
-                    "href": "http://testserver/v1/test/afvalwegingen/containers/2/",
+                    "href": "http://testserver/v1/afvalwegingen/containers/2/",
```

Comment:
Strange. This diff suggests that `test` has been removed from the href. However, when I checkout this PR, the opposite seems to be te case. Some glitch in github?